modules/administration-guide/partials/proc_using-chectl-to-install-che-on-codeready-containers.adoc

```diff
@@ -15,18 +15,18 @@ This section describes how to install {prod-short} on CodeReady Containers (CRC)
 
 .Procedure
 
-. Initiate the CRC platform:
-
-.. Configure your host machine for CodeReady Containers:
+.. Remove any previous cluster:
 +
 ----
-$ crc setup
+$ crc delete
 ----
 
-.. Remove any previous cluster:
+. Initiate the CRC platform:
```

Comment:
Hmm... this is an issue for me. (I'm the primary docs writer for CodeReady Containers/OpenShift Local, FWIW.)

From a documentation perspective, we should not use the CRC acronym. Always use the full name, CodeReady Containers. Unfortunately, we also happen to be going through a rename process (to "CRC" -- not "CodeReady Containers" -- upstream, "OpenShift Local" downstream) at the moment, so the name CodeReady Containers will not last much longer.

For this reason, perhaps we should consider replacing *all* occurrences of "CRC" with "CodeReady Containers". This would allow the team (in the future) to replace the existing content with the new name more easily (using `sed`, perhaps?).

Having the same thing referred to with multiple names (or abbreviations and acronyms) makes this task more difficult.

Reply:
TL;DR: My recommendation at the moment is simply to use the full name, CodeReady Containers.

Reply:
```suggestion
. Initiate the CodeReady Containers platform:
```